src/api/app/validators/scm_webhook_event_validator.rb

```diff
@@ -0,0 +1,46 @@
+class ScmWebhookEventValidator < ActiveModel::Validator
```

Comment:
`SCMWebhookEventValidator`

Reply:
Same comment as the other regarding the case of _SCM_. I won't change this.